.github/ISSUE_TEMPLATE/BUG-REPORT.yml

```
@@ -0,0 +1,57 @@
+name: Bug Report
+description: File a bug report.
+title: "[Bug] <title>"
```

```suggestion
title: "<title>"
```

DRY, the bug is already expressed in `type`, no need to repeat in title the scope.

---

It's a ux thing recommended by Github because it gives a clear indicator of what goes into the input field, otherwise only an empty field is displayed.  Another option might be to add  placeholder text inside the input field.  Say, something like "bug title goes here" or some such language as a hint to the contributor. Thoughts @jenna-winkler ?

---

I think we should go with 
title: <Brief summary of the issue>